### PR TITLE
[TRAFODION-28] Copyright header changes part 2

### PR DESCRIPTION
**Pull request by sandhyasun**
This set corrects the copyrights in some leftover core directories, dcs and core/rest and test files.
See Jira for more details.

### CODE REVIEW

File: core/conn/unixodbc/odbc/odbcclient/unixcli/TCPIPV4/TCPIPV4.RC [hunk 1]
```diff
@@ -45,11 +45,11 @@ BEGIN
         BLOCK "000004b0"
         BEGIN
             VALUE "Comments", "T7968G10_01NOV2004_ABC_IPV4_1220\0"
-            VALUE "CompanyName", "Hewlett-Packard Company\0"
+            VALUE "CompanyName", "Esgyn Corporation\0"
             VALUE "FileDescription", "NonStop� ODBC/MX TCPIPV4 DLL\0"
```
Comment (DaveBirdsall) [inline core/conn/unixodbc/odbc/odbcclient/unixcli/TCPIPV4/TCPIPV4.RC]: I'm wondering what the significance of "CompanyName" is. Perhaps we should put Apache Software Foundation here? Also, I'm wondering if we should take out "NonStop ODBC/MX" and replace it with something like "Apache Trafodion ODBC"?

File: core/dbsecurity/scripts/sqcertgen [hunk 2]
```diff
@@ -108,7 +108,7 @@ stateOrProvinceName_default     = California
 localityName                    = Locality Name (eg, city)
 localityName_default            = Cupertino
 0.organizationName              = Organization Name (eg, company)
-0.organizationName_default      = Hewlett-Packard Company
+0.organizationName_default      = Esgyn Corporation
```
Comment (DaveBirdsall) [inline core/dbsecurity/scripts/sqcertgen]: Similar comment

File: core/rest/src/main/java/org/trafodion/rest/TransactionsResource.java [hunk 3]
```diff
@@ -62,8 +69,26 @@
 	}
 
 	/**
-	 * Constructor
-	 * @throws IOException
+* @@@ START COPYRIGHT @@@
```
Comment (DaveBirdsall) [inline core/rest/src/main/java/org/trafodion/rest/TransactionsResource.java]: Looks like we accidentally put the copyright into this file twice?

File: core/sqf/sqenvcom.sh [hunk 4]
```diff
@@ -29,8 +29,8 @@
 #  * Add an entry in sqf/LocalSettingsTemplate.sh to make path configurable.
 ##############################################################
 
-#Product version (Trafodion or derivative product)
-export TRAFODION_VER_PROD="Apache Trafodion "
+#Product version (Trafodion, Enterprise or Enterprise Adv)
```
Comment (DaveBirdsall) [inline core/sqf/sqenvcom.sh]: "Enterprise", "Enterprise Adv" sound like product names. Just curious why we changed this away from "deriviative product"?

File: core/sql/regress/newregr/mvs/BIG/EXPECTEDMV215C [hunk 5]
```diff
@@ -20,21 +20,6 @@
 -- @@@ END COPYRIGHT @@@
 -- @@@ START COPYRIGHT @@@
```
Comment (DaveBirdsall) [inline core/sql/regress/newregr/mvs/BIG/EXPECTEDMV215C]: Did the copyright go missing in the new version?

**Comment by DaveBirdsall** [comment]
Aside from the line comments/questions I made above, it all looks good.

File: core/sql/regress/newregr/mvs/BIG/EXPECTEDMV215C (hunk 5 above)
Comment (sandhyasun) [inline core/sql/regress/newregr/mvs/BIG/EXPECTEDMV215C]: It is still there. Will remove the erroneous @@@ START COPYRIGHT sign. The newregr directory has some missing copyrights too. Will fix on next pass.

File: core/sqf/sqenvcom.sh (hunk 4 above)
Comment (sandhyasun) [inline core/sqf/sqenvcom.sh]: Merge conflict problem ! I had indeed corrected it in the last review. WIll revert to "derivative product".

File: core/rest/src/main/java/org/trafodion/rest/TransactionsResource.java (hunk 3 above)
Comment (sandhyasun) [inline core/rest/src/main/java/org/trafodion/rest/TransactionsResource.java]: Yes - looks like the script replaced those "comment" lines ! Corrected this.

File: core/dbsecurity/scripts/sqcertgen (hunk 2 above)
Comment (sandhyasun) [inline core/dbsecurity/scripts/sqcertgen]: Will make the company name and location name generic since the customer needs to create the certificate file.

**Pull request merged**